### PR TITLE
Welcome message for new members

### PR DESCRIPTION
**Pull request by isuruf**

### CODE REVIEW

File: conda_forge_webservices/update_teams.py [hunk 1]
```diff
@@ -4,18 +4,78 @@
...
+
+
+def filter_members(members):
+    out = get_filter_out_members()
+    return [m for m in members if m not in out]
+
+
+def get_handles(members):
+    mem = ['@' + m for m in filter_members(members)]
+    return ','.join(mem)
```
Comment (jakirkham) [inline conda_forge_webservices/update_teams.py]: Maybe add a space after `,`

**Comment by jakirkham** [comment]
So I've gone back and forth on this. Would be good to hear your thoughts. Namely between commenting on a commit vs. commenting on a PR. It's easier from a programmatic standpoint to do the former, but it is a little less accessible. OTOH it is a little more accessible to comment in PRs, but it is a bit more challenging to approach from a programmatic standpoint. There is the extra caveat that if the change was pushed directly (not in a PR) there is no PR to comment on. This gets a bit more interesting when considering `staged-recipes` additions where the comment may appear either in the feedstock or the staged-recipes PR depending on the approach. Anyways what do you think about all of this?

**Comment by isuruf** [comment]
> It's easier from a programmatic standpoint to do the former, but it is a little less accessible.

Commenting on a commit will appear in the PR, right? I think this is good enough for now and if somebody wants they can change it.

> This gets a bit more interesting when considering staged-recipes additions where the comment may appear either in the feedstock or the staged-recipes PR depending on the approach

Currently, this code is not triggered in the first commit of a feedstock because members are added in the staged-recipes script. I intend to turn that off so that updating team happens only here. (Or we can have staged-recipes script do the commenting on the staged-recipes PR)

**Comment by isuruf** [comment]
I'm going to merge this and play around with it a little bit.

**Comment by jakirkham** [comment]
SGTM. Are you still using that branch or is it ok to remove?

**Comment by isuruf** [comment]
Deleted it

**Comment by ericdill** [comment]
This is awesome! Thanks for the effort here @isuruf and @jakirkham

**Pull request merged**